lib/api/multiObjectDelete.js

```diff
@@ -455,6 +486,11 @@ function multiObjectDelete(authInfo, request, log, callback) {
                         });
                         return next(errors.InternalError);
                     }
+                     // Convert authorization results into an easier to handle format
+                    const actionImplicitDenies = authorizationResults.reduce((acc, curr, idx) => {
+                        const apiMethod = requestContextParams[idx].apiMethod;
+                        return Object.assign({}, acc, { [apiMethod]: curr.isImplicit });
+                    }, {});
```

Comment:
How about this?

```suggestion
                     // Convert authorization results into an easier to handle format
                     const actionImplicitDenies = authorizationResults.reduce((acc, curr, idx) => {
                        const apiMethod = requestContextParams[idx].apiMethod;
                        acc[apiMethod] = curr.isImplicit;
                        return acc;
                     }, {});
```

To avoid re-creating new objects everytime (this API is very cpu-sensitive as per my previous observations, and often used by 3rd party services like veeam)

Reply:
(not tested)